### PR TITLE
Clarify ec task usage in this repo

### PR DESCRIPTION
**Pull request by lcarva**
# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: task/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 ---
+# IMPORTANT: This Task definition exists in this repository as a means to facilitate integration
+# with RHTAP. It is NOT meant to be used in Konflux. If you are debugging EC failures in Konflux,
+# this is NOT the file you are looking for. Do NOT directly modify this file. Any change should
+# first be done in the https://github.com/enterprise-contract/ec-cli repository then synced to this
+# repository. Any pull request that modifies anything other than this comment in this file and is
+# not a sync from the ec-cli repository will be immediately closed.
```
Comment (simonbaird) [inline task/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml:7]: Could consider including the direct link https://github.com/enterprise-contract/ec-cli/tree/main/tasks/verify-enterprise-contract/0.1 or similar.

**Review by simonbaird — APPROVED** [review]

File: task/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml (hunk 1 above)
Comment (lcarva) [reply task/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml]: Done.

**Pull request merged**